image_cropping/config.py

```diff
@@ -1,7 +1,5 @@
 from appconf import AppConf
 
-from django.conf import settings
-
 
 class ImageCroppingAppConf(AppConf):
     JQUERY_URL = 'https://ajax.googleapis.com/ajax/libs/jquery/1.11.3/jquery.min.js'
```

Comment:
@anrie sollten wir hier vielleicht nicht das auch zumindest aktualisieren auf Django 2.2+ Stand? Oder gar es irgendwie aus Django (anhand Version ?) lesen?

Reply:
Auf jeden Fall! I wonder if we should even remove the option to define jQuery and always use the version provided by Django (as long as it is compatible with Jcrop).

Reply:
We could leave it adjustable in the settings, but I will remove it from the config here